### PR TITLE
PR: Algo Merge sort in C programming

### PR DESCRIPTION
**Pull request by Ravindu007**
Thia PR contains Merge sort in C programming

### CODE REVIEW

File: 2-algorithms/0-c-programming/merge_sort.c
Comment (Abishethvarman) [inline 2-algorithms/0-c-programming/merge_sort.c:1]: make a readme file, not a c implementation. Refer this https://github.com/Abishethvarman/programming-resources/blob/main/2-algorithms/0-c-programming/2-binary-search.MD

**Review by Abishethvarman — CHANGES_REQUESTED** [review]

**Comment by Ravindu007** [comment]
Sure aiya I will

**Comment by Ravindu007** [comment]
Added .md file on merge sort

File: 2-algorithms/0-c-programming/merge_sort.c
Comment (Ravindu007) [reply 2-algorithms/0-c-programming/merge_sort.c:1]: Made the requested changes

**Review by Abishethvarman — APPROVED** [review]
Approved

**Review by Thevapriyan07 — APPROVED** [review]
Approve to merge

**Pull request merged**